src/Enzo/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+
+include(charm)
+addCharmModule( "enzo" )
+
+file(GLOB SRC_FILES enzo*.cpp *F *enzo*hpp fortran.h fortran_types.h FofLib.cpp)
+
+# TODO(pgrete) Check if that file can be removed for good. It also not being used in the SCons build.
+# -> PPML_sub.F:(.text+0x5ebf): multiple definition of `qdd6_'; CMakeFiles/enzo-e.dir/PPML_QDD6.F.o:PPML_QDD6.F:(.text+0x0): first defined here
+list(FILTER SRC_FILES EXCLUDE REGEX "PPML_QDD6")
```

Comment:
Maybe make an issue for this?

Reply:
https://github.com/enzo-project/enzo-e/issues/184